apps/vanilla-three/index.html

```diff
@@ -0,0 +1,34 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <link rel="icon" type="image/svg+xml" href="/vite.svg" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <title>Vanilla JS + three.js</title>
+    <style>
```

Comment:
Remove style from here. Add it on an .css file that it's imported in main.jsx if necessary.

Reply:
I will add css ref directly in html, I would like to make plain stuff as less depenedent on tooling as possible.